docs/source/tutorial_basic.rst

```diff
@@ -246,7 +243,7 @@ Validation and Testing
 During validation and testing, the 'labels' field is removed from the observation dict.
 This tells the agent not to use these labels for training--however, the labels are
 still available via the 'eval_labels' field in case you need to compute model-side
-metrics such as perplexity.
+metrics such as loss or perplexity.
```

Comment:
'loss' is too generic doesn't mean anything

Reply:
removed